tests/ovs_test.go

```diff
@@ -35,4 +38,52 @@ var _ = Describe("ovs-cni", func() {
 			})
 		})
 	})
+
+	Describe("pod ovs-bridge connectivity tests", func() {
+		bridgeName := "br-test"
+		Context("When creating ovs-bridge on the host node", func() {
+			BeforeEach( func() {
+				err := cluster.AddOvsBridgeOnNode(bridgeName)
+				ExpectWithOffset(1, err).ToNot(HaveOccurred(), "should succeed adding ovs bridge on node")
```

Comment:
can you explain why to use `ExpectWithOffset` in the outer context as well (or at all)?

Reply:
in this test suite I have 3 modules (cluster, node, pod) with helper functions.
Following the talk with @EdDev about how make tests better and more readable, we said that the helper should not use Expect, and instead return an error (preferably with Wrap). 
`ExpectWithOffset` is like regular `Expect` but it refers to the last function (where 1 the stack offset, since I want the Expect to refer  to `AddOvsBridgeOnNode` and not a more internal function in it)

Using `ExpectWithOffset`, I can avoid using `Expect` on the helper functions, and still make sure the the Expect relates to them.

Reply:
Got it.  Thanks